### PR TITLE
Feature:  GraphQL API Date range filter for Events 

### PR DESCRIPTION
**Pull request by dip00dip**
as a part of GraphQL API Workshop

### CODE REVIEW

File: app/graphql/api/queries/events.rb [hunk 1]
```diff
@@ -14,6 +14,13 @@ def resolve(pagination: {}, filter: {})
         events = Event.display.published
         events = events.where(status: filter[:status]) if filter[:status].present?
 
+        if filter[:start_at].present?
+          start_from = filter[:start_at][:from] || 10.years.ago #DateTime.strptime("0", "%s")
+          end_at = filter[:start_at][:to] || 10.years.from_now
```
Comment (sveredyuk) [inline app/graphql/api/queries/events.rb]: `end_at = filter.dig(:start_at, :to) || Time.curent`

**Comment by sveredyuk** [comment]
I think we can merge this if @dip00dip resolve conflicts and pass `rubocop -a` ;)

File: app/graphql/api/queries/events.rb (hunk 1 above)
Comment (dip00dip) [reply app/graphql/api/queries/events.rb]: changed.

**Comment by dip00dip** [comment]
rebased and forced all check pass :)

**Review by sveredyuk — APPROVED** [review]
Thanks, man @dip00dip

**Pull request merged**